contrib/test/crio-integration-playbook.yaml

```diff
@@ -215,31 +215,31 @@
     args:
       chdir: /root/src/github.com/kubernetes-incubator/cri-o/
   - name: run integration tests RHEL
-    shell: 'CGROUP_MANAGER=systemd STORAGE_OPTS="--storage-driver=overlay2 --storage-opt overlay2.override_kernel_check=1" make localintegration 2>&1 > testout.txt'
+    shell: 'CGROUP_MANAGER=cgroupfs STORAGE_OPTS="--storage-driver=overlay2 --storage-opt overlay2.override_kernel_check=1" make localintegration 2>&1 > testout.txt'
```

Comment:
I would like to point out the reason why `runc` doesn't use `systemd` by default anymore is because Mrunal, Michael and mysellf all found so many different issues with `systemd` related to cgroups. Note that `cgroupfs` will still set up `/sys/fs/cgroup/systemd/...` so I would be interested in knowing why you need to run `runc` with `systemd` cgroup management...